Documentation/ceph-csi-drivers.md

```diff
@@ -108,105 +121,110 @@ Once the plugin is successfully deployed, test it by running the following examp
+```
 
 ```bash
-kubectl exec -ti -n rook-ceph rook-ceph-operator-6c49994c4f-pwqcx /bin/sh
-sh-4.2# ceph auth ls
-installed auth entries:
+cat << EOF | kubectl create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: csi-rbd-secret
+  namespace: default
+data:
+  # Key value corresponds to a user name defined in Ceph cluster
+  admin: ${admin_key}
+  # Key value corresponds to a user name defined in Ceph cluster
+  kubernetes: ${kubernetes_key}
+  # if monValueFromSecret is set to "monitors", uncomment the
+  # following and set the mon there
+  #monitors: BASE64-ENCODED-Comma-Delimited-Mons
+EOF
+```
 
-osd.0
-	key: AQA3pa1cN/fODBAAc/jIm5IQDClm+dmekSmSlg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.1
-	key: AQBXpa1cTjuYNRAAkohlInoYAa6A3odTRDhnAg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.2
-	key: AQB4pa1cvJidLRAALZyAtuOwArO8JZfy7Y5pFg==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-osd.3
-	key: AQCcpa1cFFQRHRAALBYhqO3m0FRA9pxTOFT2eQ==
-	caps: [mgr] allow profile osd
-	caps: [mon] allow profile osd
-	caps: [osd] allow *
-client.admin
-	key: AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==
-	auid: 0
-	caps: [mds] allow *
-	caps: [mgr] allow *
-	caps: [mon] allow *
-	caps: [osd] allow *
-client.bootstrap-mds
-	key: AQD6pK1crJyZCxAA1UTGwtyFv3YYFcBmhWHyoQ==
-	caps: [mon] allow profile bootstrap-mds
-client.bootstrap-mgr
-	key: AQD6pK1c2KaZCxAATWi/I3i0/XEesSipy/HeIA==
-	caps: [mon] allow profile bootstrap-mgr
-client.bootstrap-osd
-	key: AQD6pK1cwa+ZCxAA7XKXRyLQpaHZ+lRXeUk8xQ==
-	caps: [mon] allow profile bootstrap-osd
-client.bootstrap-rbd
-	key: AQD6pK1cULmZCxAA4++Ch/iRKa52297/rbHP+w==
-	caps: [mon] allow profile bootstrap-rbd
-client.bootstrap-rgw
-	key: AQD6pK1cbMKZCxAAGKj5HaMoEl41LHqEafcfPA==
-	caps: [mon] allow profile bootstrap-rgw
-mgr.a
-	key: AQAZpa1chl+DAhAAYyolLBrkht+0sH0HljkFIg==
-	caps: [mds] allow *
-	caps: [mon] allow *
-	caps: [osd] allow *
+## Create RBD StorageClass
 
-#encode admin/user key
-sh-4.2# echo -n AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==|base64
-QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
-#or
-sh-4.2# ceph auth get-key client.admin|base64
-QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
-```
+Create a ``storage-class.yaml`` file, be sure to update the ``monitors`` entry to reflect the IP of your
+monitors.  Using our example output above, collect the monitor IP addresses  (``kubectl -n rook-ceph get all``)
+
+*NOTE*
+Our example below uses the ``rbd`` pool we created in our first step, if you used a different name or are adding an additional pool,
+be sure to modify it in the storage class 
+
+```bash
+cat << EOF | kubectl create -f -
```

Comment:
In documentation I think it works better to have the contents that they save to the yaml file, then use the `kubectl create` command with the pattern that was used previously. But it's probably not worth changing since this is a short-lived doc once full integration is done with rook.